### PR TITLE
TP-963: Update Gigaspaces to 16.1.0 and Spring Framework to 5.3.7

### PR DESCRIPTION
**Pull request by ath0s**
Also includes some cleanup of dependencies, mostly regarding logging.

### CODE REVIEW

File: examples/lunch-web/pom.xml [hunk 1]
```diff
@@ -101,78 +95,10 @@
...
-			</dependency>
-			<dependency>
-				<groupId>${spring.groupId}</groupId>
-				<artifactId>spring-tx</artifactId>
-				<version>${spring.version}</version>
-			</dependency>
-			<dependency>
-				<groupId>${spring.groupId}</groupId>
-				<artifactId>spring-aspects</artifactId>
-				<version>${spring.version}</version>
```
Comment (pativa) [inline examples/lunch-web/pom.xml:175]: Much better using the spring boot bom instead of this long list of dependencies! 👍

**Review by pativa — APPROVED** [review]
Nice spring cleaning!

File: examples/lunch-web/pom.xml (hunk 1 above)
Comment (ath0s) [reply examples/lunch-web/pom.xml:175]: Yes 😄

**Review by Bjornelmers — APPROVED** [review]
Very good!

**Pull request merged**